task/handler/log/api/src/main/java/io/knotx/fragments/task/handler/log/api/model/GraphNodeResponseLog.java

```diff
@@ -65,16 +66,25 @@ public GraphNodeResponseLog setTransition(String transition) {
   }
 
   /**
-   * List of node invocation(s) logs.
+   * Node response log.
    *
-   * @return list of node invocation(s) logs
+   * @return node response log
    */
-  public JsonArray getInvocations() {
-    return invocations;
+  public JsonObject getLog() {
+    return log;
   }
 
-  public GraphNodeResponseLog setInvocations(JsonArray invocations) {
-    this.invocations = invocations;
+  public GraphNodeResponseLog setLog(JsonObject log) {
+    this.log = log;
+    return this;
+  }
+
+  public List<GraphNodeErrorLog> getErrors() {
+    return errors;
+  }
+
+  public GraphNodeResponseLog setErrors(List<GraphNodeErrorLog> errors) {
```

Comment:
Not used? Perhaps should be called in `GraphNodeResponseLog.newInstance`

Reply:
Fixed.